index.html

```diff
@@ -1,11 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
   <title>Challenge 1</title>
 </head>
+
 <body>
-  <h1>Challenge 1</h1>
+  <h1>COMPUMUNDO HIPERMEGA RED</h1>
+
+  <main>
+    <div class="heading-line">
+      <hr class="heading-hr">
+    </div>
+
+    <section class="main-charter">
+      <div class="headingimagebox">
+        <img width="500" src="https://www.notebookcheck.org/fileadmin/Notebooks/News/_nc3/Zephyrus_Duo93.jpg" alt="Asus Zephyrus">
+      </div>
+      <div class="headingbox">
+        <div class="featuredcircle">
+          <img width="50" src="https://www.pnguniverse.com/wp-content/uploads/2020/10/Homero-Simpson-1024x490.png" alt="Homero Simpson" id="SL">
+          <p><strong>Featured by</strong></p>
+        </div>
+        <div class="businessbox">
+          <div class="circle">
+            <p>Business</p>
+          </div>
+          <div class="date">
+            <p>August 19, 2023</p>
+          </div>
+        </div>
+
+        <h2>COMPUMUNDO HIPERMEGA RED presenta</h2>
+        <p>"Esta industria avanza tan rápido que es difícil saber"</p>
+
+        <div class="headingboxby">
+          <div class="headinginnerbox">
+            <img width="50" src="https://www.dibujos-faciles.com/wp-content/uploads/2022/12/dibujo-de-Homer-Simpson-paso7-1.jpg" alt="Homero Simpson">
+          </div>
+          <div class="headinginnerbox2">
+            <p>by <strong>Homero Simpson</strong></p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="hr-charter">
+      <div class="separator-box">
+        <div class="separator-text">
+          <p><strong>Trending now</strong></p>
+        </div>
+        <div class="separator-line">
+          <hr class="styled-hr">
+        </div>
+      </div>
+    </section>
+
+    <section class="sub-charter">
+      <div class="card">
+        <div class="img">
+          <img class="imgf" src="https://i5.walmartimages.com/asr/d8541d9e-f61b-498e-8c26-89a0bad39402.3b417dea36b02392a75882aff3b2b102.jpeg" alt="Asus ZenBook 14X Oled">
+        </div>
+        <div class="marketingbox">
+          <div class="circle">
+            <p>Business</p>
+          </div>
+          <div class="date">
+            <p>August 19, 2023</p>
+          </div>
+        </div>
+
+        <h3>Asus ZenBook 14X Oled (UX3404)</h3>
+        <p><b>COLOR:</b> Inkwell Gray<br>
```

Comment:
Acá pueden utilizar un párrafo para cada línea
```suggestion
        <p><b>COLOR:</b> Inkwell Gray<br><p/>
```
Y así sucesivamente